css/css-masking/clip-path/reference/clip-path-shape-003-ref.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Masking: Test clip-path property and shape function with nonzero fill</title>
```

Comment:
nit: Perhaps using "Reference of xxx" would be better.

Reply:
Fixed